website/docs/r/cloudant.html.markdown

```diff
@@ -0,0 +1,136 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_cloudant"
+description: |-
+  Manages Cloudant instance.
+subcategory: "Cloudant"
+---
+
+# ibm_cloudant
+
+Provides a resource for IBM Cloudant. This allows an IBM Cloudant service instance to be created, updated, or deleted.
+For more information, about how to use this service, see the official [Cloudant documentation](https://cloud.ibm.com/docs/Cloudant?topic=Cloudant-getting-started-with-cloudant).
+
+## Example Usage
+
+```hcl
+resource "ibm_cloudant" "cloudant" {
+  name     = "cloudant-service-name"
+  location = "us-south"
+  plan     = "standard"
+
+  legacy_credentials  = true
+  include_data_events = false
+  capacity            = 1
+  enable_cors         = true
+
+  cors_config {
+    allow_credentials = false
+    origins           = ["https://example.com"]
+  }
+
+  timeouts {
+    create = "15m"
+    update = "15m"
+    delete = "15m"
+  }
+}
+```
+
+## Timeouts
+
+ibm_cloudant provides the following [Timeouts](https://www.terraform.io/docs/language/resources/syntax.html#operation-timeouts) 
+configuration options:
+
+* `create` - (Default 10 minutes) The creation of the IBM Cloudant instance is considered failed if no response received.
+* `delete` - (Default 10 minutes) The update of the IBM Cloudant instance is considered failed if no response received.
+* `update` - (Default 10 minutes) The deletion of the IBM Cloudant instance is considered failed if no response received.
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required, String) A name for the resource instance.
+* `location` - (Required, Forces new resource, String) Target location or environment to create the resource instance.
+* `plan` - (Required, String) The plan type of the service.
+* `capacity` - (Optional, Number) A number of blocks of throughput units. A block consists of 100 reads/sec, 50 writes/sec, and 5 global queries/sec of provisioned throughput capacity.
+  * Constraints: The default value is `1`. Setting capacity is not supported for lite plan.
```

Comment:
> Setting capacity is not supported for lite plan.

If I remember correctly it is, but cannot set to another value than 1.

Reply:
I think this is the same thing, just playing with words. Like: `half full or half empty glass`

Reply:
hmm I see actually the value we pass to the RC is `lite` so maybe we should leave the casing, but just put it in backticks.

Reply:
I changed it to `lite`: 0eafc34f